### ISSUE TITLE
Improve colour contrast to achieve AA rating

### ISSUE THREAD

**Issue by dan2k3k4** [karlhorky/karlhorky.github.io#2]
Using http://www.checkmycolours.com/ - I submitted the URL https://work.karlhorky.com/ which returned 74 errors for contrasting colours.

Testing done on 74 elements
- Luminosity Contrast Ratio: 19 failures 
- Brightness difference: 19 failures
- Color difference: 33 failures

**Comment by dan2k3k4** [comment]
Came across http://www.colorbox.io/ which might help to generate a colour scheme that is contract compliant.
